src/app/pages/TransactionDetailPage/CurrentFiatValue.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { FiatMoneyAmountBox } from '../../components/Account'
+import Tooltip from '@mui/material/Tooltip'
+import { CoinGeckoReferral } from '../../components/CoinGeckoReferral'
+import HelpIcon from '@mui/icons-material/Help'
+import { TokenPriceInfo } from '../../../coin-gecko/api'
+
+type CurrentFiatValueProps = Pick<TokenPriceInfo, 'price' | 'hasUsedCoinGecko'> & {
+  amount: number
+}
+
+export const CurrentFiatValue: FC<CurrentFiatValueProps> = ({ amount, price, hasUsedCoinGecko }) => {
+  const { t } = useTranslation()
+  return price === undefined ? null : (
+    <FiatMoneyAmountBox>
+      <span>
+        {t('common.fiatValueInUSD', {
+          value: amount * price,
+          formatParams: {
+            value: {
+              currency: 'USD',
+            } satisfies Intl.NumberFormatOptions,
+          },
```

Comment:
Just noticed that this sometimes rounds up!
https://explorer.dev.oasis.io/mainnet/sapphire/transactions/0x0de151f3bc5bb385526933b075bd7063733809a815e4ce3fbd428b485f955e15

0.100000001686272452 ROSE * 0.05003 => $0.01

Reply:
And `roundingMode: 'trunc',` doesn't work in firefox

Reply:
How about saying "**Approximate** fiat value", instead of "Fiat value"?